### DISCUSSION TITLE
how to add buttons like Events and stuff to a GDPS hosted on this?

### DISCUSSION THREAD

**Discussion by jmxamongusmodder** [Cvolton/GMDprivateServer#918] [Q&A]
how to add buttons like Events and stuff to a GDPS hosted on this?

**Comment by MegaSa1nt** [comment]
wait for 2.2

**Reply by ghost** [reply]
Thats why all 2.2 GDPS's exist.

**Comment by Kingminer7** [comment]
mod the apk/exe if you know how, or use the lite/meltdown/world/subzero apks
and figure out how to add stuff to those

**Reply by ghost** [reply]
There are tutorials on how to mod GD on YouTube, and other websites.
